### PR TITLE
Initial post for grafana-polystat-panel

### PR DESCRIPTION
**Pull request by briangann**
There's some more content to add, but this has the bulk of the post.

### CODE REVIEW

File: content/blog/2019-03-31-using-grafana-polystat-panel-plugin.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+cover_opacity = "0.6"
+cover_blur = "1px"
+description = "Part 1 of the the grafana-polystat-panel plugin tutorial."
+categories = ["Tutorials", "Grafana", "Plugins","Prometheus"]
+excerpt = ""
++++
+
+## Polystat
+
+The grafana-polystat-panel [plugin](https://grafana.com/plugins/grafana-polystat-panel) was created to provide a way to roll up multiple metrics and implement flexible drill downs to other dashboards.
```
Comment (tomwilkie) [inline content/blog/2019-03-31-using-grafana-polystat-panel-plugin.md]:
```suggestion
The [grafana-polystat-panel plugin](https://grafana.com/plugins/grafana-polystat-panel) was created to provide a way to roll up multiple metrics and implement flexible drill downs to other dashboards.
```

(Its better is hyperlinks are entire phrases, helps old school SEO)

**Comment by tomwilkie** [comment]
Looks good Brian!  Feedback:
- The balance of explanatory text to screen shots & queries seems a bit off - on the side of too little text.  It makes it hard for me see the narrative here.  You mentioned there is more content to add, so perhaps that'll fix this.
- Why do you roll all the nodes up into one hexagon at the end?  I would have thought a hexagon per node (with all the per-node stats rolled into one) would have been more useful to pick up misbehaving nodes.
- Perhaps starting the post with the animation/finished article would make it more obvious what you're trying to build and hook people into the story?  Would also help to motivate the whole post with some more narrative - building a dashboard to help oncall engineers dig into Cassandra behaviour and quickly identify misbehaving node?  Maybe make reference to USE method and apply some of that logic here?

File: content/blog/2019-03-31-using-grafana-polystat-panel-plugin.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
++++
+title = "Grafana Plugin: Polystat Panel (Part 1)"
+author = "brian"
+date = "2019-04-01"
+keywords = ["Plugins","Prometheus"]
+cover_image = "/assets/img/grafana_logo_transparent.png"
+cover_opacity = "0.6"
+cover_blur = "1px"
+description = "Part 1 of the the grafana-polystat-panel plugin tutorial."
+categories = ["Tutorials", "Grafana", "Plugins","Prometheus"]
```
Comment (tomwilkie) [inline content/blog/2019-03-31-using-grafana-polystat-panel-plugin.md]: I'd flip this round; less categories, more keyword.s

**Comment by briangann** [comment]
Excellent feedback @tomwilkie I'll get this cleaned up today. I did mean to have a final result as you mentioned, and starting with showing the goal would definitely make it flow better (have a story).

Thanks!

Brian

**Pull request merged**